google/resource_google_organization_policy.go

```diff
@@ -99,6 +114,56 @@ var schemaOrganizationPolicy = map[string]*schema.Schema{
 		Type:     schema.TypeString,
 		Computed: true,
 	},
+	// Detect changes to local file or changes made outside of Terraform to the file stored on the server.
+	"detect_policy_change": &schema.Schema{
```

Comment:
I'm relatively new to Terraform, but I haven't seen this pattern before.  Is this something that you've seen done elsewhere?